partitura/utils/music.py

```diff
@@ -1289,19 +1430,22 @@ def slice_notearray_by_time(note_array, start_time, end_time, time_unit='auto',
     if clip_onset_duration and len(active_idx) > 0:
         psi = np.where(note_array_slice[onset_unit] < start_time)[0]
         note_array_slice[psi] = start_time
-        adj_offsets = np.clip(note_array_slice[onset_unit] +
-                              note_array_slice[duration_unit],
-                              a_min=None,
-                              a_max=end_time)
-        note_array_slice[duration_unit] = (adj_offsets -
-                                           note_array_slice[onset_unit])
+        adj_offsets = np.clip(
+            note_array_slice[onset_unit] + note_array_slice[duration_unit],
+            a_min=None,
+            a_max=end_time,
+        )
+        note_array_slice[duration_unit] = adj_offsets - note_array_slice[onset_unit]
 
     return note_array_slice
 
 
-def note_array_from_part(part, include_pitch_spelling=False,
-                         include_key_signature=False,
-                         include_time_signature=False):
+def note_array_from_part(
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.